recon/src/sha256a.rs

```diff
@@ -214,8 +209,15 @@ impl Key for Bytes {
     }
 
     fn max_value() -> Self {
-        // We need a value that sorts greater than any other byte slice.
-        // As this is test specific code we can use a sufficiently large 0xFF byte array.
+        // We need a value that sorts greater than any multiformat.
+        // Multiformats start with a multiformats.varint
+        // (https://github.com/multiformats/unsigned-varint)
+        // the max_value for a multiformat varint is 0x7fffffff_ffffffff (2^63-1)
+        // multiformats.varint.encode(0x7fffffff_ffffffff)
+        // b'\xff\xff\xff\xff\xff\xff\xff\xff\x7f'
+        // therefore all multiformat data will be less then
+        // b'\xff\xff\xff\xff\xff\xff\xff\xff\x80'
+        // [0xFF, 9]
         [0xFF; 1024].as_slice().into()
```

Comment:
looks like a small typo here, the last line should be deleted and the last comment uncommented.